### PR TITLE
Update Winbox to 3.36

### PR DESCRIPTION
**Pull request by juanpedrojose**
Update Winbox x64 executable to 3.36

### CODE REVIEW

**Comment by nrlquaker** [comment]
Thanks for the PR, @juanpedrojose. Please update winbox version in the README.md.

File: CHANGELOG_WINBOX.md [hunk 1]
```diff
@@ -219,4 +237,4 @@
 - make file copy & paste work between routers again;
 - make Enter key activate Connect if one of connect fields were active even when master password field has not been entered yet;
 - fixed Reconnect in RoMON mode with "Open In New Window" enabled;
-- other fixes.
\ No newline at end of file
+- other fixes.
```
Comment (nrlquaker) [inline CHANGELOG_WINBOX.md:240]: Please revert this change.

File: CHANGELOG_WINBOX.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
 # Winbox changelog
 
+## 3.36
+
+- fix issue with non existing records still in winbox ros7
```
Comment (nrlquaker) [inline CHANGELOG_WINBOX.md:5]: Please add `;` ad the end of lines and `.` in the end of last line.

Comment (juanpedrojose) [reply CHANGELOG_WINBOX.md]: Done.

File: CHANGELOG_WINBOX.md (hunk 1 above)
Comment (juanpedrojose) [reply CHANGELOG_WINBOX.md:239]: I see that this line has changed but I don't see the change itself.
I took that file from the `master` branch, added information about 3.36 but same result.

```
-- other fixes.
+- other fixes.
```

Any advice?

Comment (nrlquaker) [reply CHANGELOG_WINBOX.md:239]: Seems like line ending problem. Anyway you can keep this change, because there were no empty line in the end of file.

**Comment by nrlquaker** [comment]
Thanks for contribution!
